cms/djangoapps/contentstore/views/course.py

```diff
@@ -1367,12 +1389,21 @@ def group_configurations_list_handler(request, course_key_string):
         if 'text/html' in request.META.get('HTTP_ACCEPT', 'text/html'):
             group_configuration_url = reverse_course_url('group_configurations_list_handler', course_key)
             course_outline_url = reverse_course_url('course_handler', course_key)
-            configurations = GroupConfiguration.add_usage_info(course, store)
+            should_show_experiment_groups = are_content_experiments_enabled(course)
+            if should_show_experiment_groups:
+                experiment_group_configurations = GroupConfiguration.get_split_test_partitions_with_usage(course, store)
+            else:
+                experiment_group_configurations = None
+            content_group_configuration = GroupConfiguration.get_or_create_content_group_configuration(
+                course
+            ).to_json()
```

Comment:
Understandable that you won't be able to address nitpicks, given your deadline. However, I'm still going to point out the things that I think should be changed, so that we can go back and review this pull request later on, and perhaps make a follow-up PR to fix the problems. (Heck, *I* might make a PR to fix these problems once this PR lands!)